### ISSUE TITLE
Recaptcha form component

### ISSUE THREAD

**Issue by xorock** [blade-ui-kit/blade-ui-kit#79]
Have you thought about implementing the recaptcha component? I think it is used so widely that it can be very useful.

**Comment by driesvints** [comment]
Which one? The one from Google?

**Comment by xorock** [comment]
Yes.
https://www.google.com/recaptcha/about/

**Comment by driesvints** [comment]
Yeah think it could be useful. Feel free to send in a PR if you want.

**Comment by Qoraiche** [comment]
no updates about this?
